ohai.gemspec

```diff
@@ -25,7 +25,7 @@ Gem::Specification.new do |s|
   s.add_dependency "ipaddress"
   s.add_dependency "wmi-lite", "~> 1.0"
   s.add_dependency "ffi", "~> 1.9"
-  s.add_dependency "chef-config", ">= 12.5.0.alpha.1", "< 15"
+  s.add_dependency "chef-config", ">= 12.8", "< 15"
```

Comment:
just make this `~>` the ohai major version?

Reply:
We need it to work with chef-config 14, which isn't out though. It needs 13-14 at a minimum